src/checkers/inference/solver/strategy/GraphSolvingStrategy.java

```diff
@@ -43,7 +44,7 @@
  * This solving strategy is useful when solving constraints for a type system with a huge number of qualifers.
  * Normal plain solving strategy meet exponentially increased solving time in this case.
  */
-public class GraphSolvingStrategy extends AbstractSolvingStrategy implements SolvingStrategy {
+public class GraphSolvingStrategy extends AbstractSolvingStrategy {
```

Comment:
is `implements SolvingStrategy` still required?

Reply:
No, because the superclass implements that interface. It is redundant to list all the interfaces again.